js/runtime.js

```diff
@@ -976,4 +981,9 @@
         }
     };
 
+    function checkAllBlocksHaveRuntimeFunctions(){
```

Comment:
Yes, at first I was thinking it should be a runtime check, but then realized we probably could and should check in Travis. In any case it doesn't belong in this PR.